app/experimenter/nimbus-ui/src/components/PageHome/index.tsx

```diff
@@ -6,18 +6,61 @@ import React from "react";
 import AppLayout from "../AppLayout";
 import { RouteComponentProps, Link } from "@reach/router";
 import Head from "../Head";
+import { useQuery } from "@apollo/client";
+import { getAllExperiments_experiments } from "../../types/getAllExperiments";
+import { GET_ALL_EXPERIMENTS } from "../../gql/experiments";
+import PageLoading from "../PageLoading";
+import sortByStatus from "./sortByStatus";
+import DirectoryTable, {
+  DirectoryLiveTable,
+  DirectoryCompleteTable,
+  DirectoryDraftsTable,
+} from "../DirectoryTable";
 
 type PageHomeProps = {} & RouteComponentProps;
 
-const PageHome: React.FunctionComponent<PageHomeProps> = () => (
-  <AppLayout testid="PageHome">
-    <Head title="Experiments" />
+export const Body = () => {
+  const { data, loading } = useQuery<{
+    experiments: getAllExperiments_experiments[];
+  }>(GET_ALL_EXPERIMENTS);
```

Comment:
Two minor suggestions with this naming, not blockers: 1) we have a pattern of suffixing our queries and mutations with `_QUERY` or `_MUTATION`, 2) eventually this query may accept filtering input so "all" may not be true. This could be `GET_EXPERIMENTS_QUERY`